docs/getting-started-guides/docker.md

```diff
@@ -113,7 +113,13 @@ Now run `docker ps` you should see nginx running.  You may need to wait a few mi
 ### Expose it as a service
 
 ```shell
-kubectl expose rc nginx --port=80
+kubectl expose deployment nginx --port=80
+```
+
+If running with docker-machine, you want to expose the deployment on the docker-machine ip as well, so use
+
+```shell
+kubectl expose deployment nginx --port=80 --external-ip=$(docker-machine ip)
```

Comment:
Or possibly just delete the section that parses out the cluster ip since it doesn't work for docker-machine.